src/libraries/System.Runtime.Caching/src/System/Runtime/Caching/Configuration/CachingSectionGroup.cs

```diff
@@ -3,9 +3,13 @@
 
 using System;
 using System.Configuration;
+using System.Runtime.Versioning;
 
 namespace System.Runtime.Caching.Configuration
 {
+#if NET5_0_OR_GREATER
```

Comment:
We usually don't ifdef and just set this property (in this case in the src csproj) to include the platform attributes on downstream tfms: `<IncludePlatformAttributes>true</IncludePlatformAttributes>`

Reply:
we already set that propery, but it's not enough here because `browser` is not recognized as a valid platform on these downstream tfms and the build would fail with:

```
error CA1418: The platform 'browser' is not a known platform name
```

Reply:
@buyaa-n is that expected?

Reply:
Yes it is, maybe we should disable the CA1418 for TFMs below 5.0